---
std_msgs/Header header
geometry_msgs/Point[] points
string[] link_names
geometry_msgs/Vector3 gripper1_force
geometry_msgs/Vector3 gripper2_force
geometry_msgs/Vector3 gripper1_velocity
geometry_msgs/Vector3 gripper2_velocity
sensor_msgs/Image camera_image